dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md

```diff
@@ -0,0 +1,214 @@
+| `tidb.service.mysqlNodePort` | The mysql `NodePort` that TiDB Service exposes |  |
+| `tidb.service.exposeStatus` | The port that indicates the expose status of TiDB Service | `true` |
+| `tidb.service.statusNodePort` | The `NodePort` exposed through specifying the status of TiDB Service |  |
+| `tidb.separateSlowLog` | Whether to run in sidebar mode the `SlowLog` of TiDB exported via independent container | If the version of TiDB Operator is v1.0.0-beta.3 or ealier, the default value is `false`.<br>If the version of TiDB Operator is later than v1.0.0-beta.3, the default value is `true`.  |
+| `tidb.slowLogTailer.image` | The image of TiDB's `slowLogTailer`. `slowLogTailer` is a container of sidecar type, used to export the `SlowLog` of TiDB. This configuration only takes effect when `tidb.separateSlowLog`=`true`. | `busybox:1.26.2` |
+| `tidb.slowLogTailer.resources.limits.cpu` | The limit on CPU resource per TiDB Pod's `slowLogTailer` | `100m` |
+| `tidb.slowLogTailer.resources.limits.memory` | The limit on memory resource per TiDB Pod's `slowLogTailer` | `50Mi` |
+| `tidb.slowLogTailer.resources.requests.cpu` | The request of each TiDB Pod's `slowLogTailer` for CPU resource | `20m` |
+| `tidb.slowLogTailer.resources.requests.memory` | The request of each TiDB Pod's `slowLogTailer` for memory resource | `5Mi` |
+| `tidb.plugin.enable` | Whether to enable TiDB plugin | `false` |
+| `tidb.plugin.directory` | This parameter specifies the directory where the TiDB plugin is located. | `/plugins` |
+| `tidb.plugin.list` | This parameter specifies a list of plugins loaded on TiDB. The naming rules of Plugin ID: plugin name-version. For example: 'conn_limit-1'. | `[]` |
+| `tidb.preparedPlanCacheEnabled` | Whether to enable TiDB's prepared plan cache<br>If the version of TiDB Operator is later than v1.0.0-beta.3, you can configure this parameter via `tidb.config`:<br>`[prepared-plan-cache]`<br>`enabled = false` | `false` |
+| `tidb.preparedPlanCacheCapacity` | The cache capacity of TiDB's prepared plan<br>If the version of TiDB Operator is later than v1.0.0-beta.3, you can configure this parameter via `tidb.config`:<br>`[prepared-plan-cache]`<br>`capacity = 100`  | `100` |
+| `tidb.txnLocalLatchesEnabled` | Whether to enable the memory lock for transaction. It is recommended to enable the lock when there are many conflicts among local transactions.<br>If the version of TiDB Operator is later than v1.0.0-beta.3, you can configure this parameter via `tidb.config`:<br>`[txn-local-latches]`<br>`enabled = false` | `false` |
+| `tidb.txnLocalLatchesCapacity` |  The capacity of the local latch of a transaction<br>If the version of TiDB Operator is later than v1.0.0-beta.3, you can configure this parameter via `tidb.config`:<br>`[txn-local-latches]`<br>`capacity = 10240000` | `10240000` |
```

Comment:
https://github.com/pingcap/docs-cn/pull/1590 reverted some changes in https://github.com/pingcap/docs-cn/pull/1582 and I have fixed them in https://github.com/pingcap/docs-cn/pull/1643, please help sync the change.

Reply:
Sure